### ISSUE TITLE
On sample web server: webProxyServer.WebServerOrchestrator is always null

### ISSUE THREAD

**Issue by danr777** [Mimetis/Dotmim.Sync#238]
I have downloaded latest project and ran samples... and the WebServerOrchestrator object inside the proxy server is always null and the handler definition of: webProxyServer.WebServerOrchestrator.OnApplyChangesFailed(e => always fails because of this..

Is this not the case normally?

**Comment by Mimetis** [comment]
This issue has been fixed with the release of the version `v0.5`.  
If you still experiencing the same bug, please comment here, I will re-open the issue

For your information, extracted from the documentation (https://mimetis.github.io/Dotmim.Sync/Web), here is a sync controller, using `v0.5`:

```cs
[Route("api/[controller]")]
[ApiController]
public class SyncController : ControllerBase
{
    private WebServerManager webServerManager;

    // Injected thanks to Dependency Injection
    public SyncController(WebServerManager webServerManager) => this.webServerManager = webServerManager;

    [HttpPost]
    public async Task Post()
    {
        try
        {
            // Get Orchestrator regarding the incoming scope name (from http context)
            var orchestrator = webServerManager.GetOrchestrator(this.HttpContext);

            orchestrator.OnApplyChangesFailed(e =>
            {
                if (e.Conflict.RemoteRow.Table.TableName == "Region")
                {
                    e.Resolution = ConflictResolution.MergeRow;
                    e.FinalRow["RegionDescription"] = "Eastern alone !";
                }
                else
                {
                    e.Resolution = ConflictResolution.ServerWins;
                }
            });

            var progress = new SynchronousProgress<ProgressArgs>(pa => Debug.WriteLine($"{pa.Context.SyncStage}\t {pa.Message}"));

            // handle request
            await webServerManager.HandleRequestAsync(this.HttpContext, default, progress);

        }
        catch (Exception ex)
        {
            await WebServerManager.WriteExceptionAsync(this.HttpContext.Response, ex);
        }
    }

    /// <summary>
    /// This Get handler is optional. It allows you to see the configuration hosted on the server
    /// The configuration is shown only if Environmenent == Development
    /// </summary>
    [HttpGet]
    public async Task Get() => await webServerManager.HandleRequestAsync(this.HttpContext);
}
```

**Comment by danr777** [comment]
Sorry didn't have a chance to check but awesome you fixed it!!
